tests/unit/test_webhook.py

```diff
@@ -455,6 +455,13 @@ def test_prepare_rejected_issue(self):
         self.assertEqual(type(actual), dict)
         self.assertEqual(actual, expected)
 
+    @patch('webcompat.webhooks.helpers.make_classification_request')
+    def test_get_issue_classification(self, mock_classification):
+        """Test that api is called only once with response code is 200."""
```

Comment:
What is happening if it's called more than once?
Does it fail? Can't happen?

Reply:
It's making sure that if we receive a 200 status, `make_classification_request` is not called again.

So basically classification in bugbug is asynchronous, the first request submits an issue for classification (in response we get 202 as the result is not ready yet) and we need to make a second one to get the results (and we get 200). 
This needs another test, but I'm not sure how to do it with a mock:
 
1 request -> 202 
2 request -> 200
assert that mock_classification called twice